### ISSUE TITLE
Other gene clusters

### ISSUE THREAD

**Issue by JPTin** [nselem/bigscape#3]
Hello, I had issues posting to git.wageningenur so I hope this gets through...

In the project overview in the wageningenur git you mention that "in principle" this program could be run on other gene clusters. Are there any examples or tutorials of this? would it require modification to the code, or a selection of parameters?

Thank you in advance.

**Comment by nselem** [comment]
Hello
Would you like to run CORASON or BiGSCAPE?

**Comment by JPTin** [comment]
I would like to run both. If I can import gbk files into bigscape can I use my own anchor files, and turn off classify (using the parameters) then import they clusters into corason as normal. Would this work in theory?

**Comment by nselem** [comment]
Why would you want to use BiGSCAPE without classifying BGCs? You can use your gbk files directly into CORASON and run a search against your genomes or other BGCs

**Comment by JPTin** [comment]
I would like to cluster my operons through BiGSCAPE first, however I could run CORASON independently as well I suppose.

**Comment by nselem** [comment]
Yes, you could run CORASON and that is also a way to obtain similar BGCs to your referebce BGC.

**Comment by JPTin** [comment]
Thank you for your quick responses :)
